### PR TITLE
Merge notifications functionality into getAccountTransactions

### PR DESCRIPTION
**Pull request by clark800**

### CODE REVIEW

File: test/api-test.js [hunk 1]
```diff
@@ -101,6 +101,17 @@ describe('RippleAPI', function() {
       _.partial(checkResult, accountTransactionsResponse, done));
   });
 
+  // TODO: this doesn't test much, just that it doesn't crash
```
Comment (lumberj) [inline test/api-test.js]: Try testing it locally at least?

File: src/api/ledger/transactions.js [hunk 2]
```diff
@@ -116,20 +122,40 @@ function getAccountTx(remote, address, options, marker, limit, callback) {
...
+    composeAsync((txs) => txs.sort(compare), callback));
+}
+
+function getAccountTransactions(address, options, callback) {
+  validate.address(address);
+  validate.getAccountTransactionsOptions(options);
+
+  const remote = this.remote;
+  if (options.start) {
+    getTransaction.bind(this)(options.start, {}, (error, tx) => {
```
Comment (lumberj) [inline src/api/ledger/transactions.js]: Why do you need `bind` here?

File: src/api/ledger/transactions.js [hunk 3]
```diff
@@ -116,20 +122,40 @@ function getAccountTx(remote, address, options, marker, limit, callback) {
         .filter((tx) => tx.validated)
         .map(parseAccountTxTransaction)
         .filter(_.partial(transactionFilter, address, options))
+        .filter(_.partial(orderFilter, options))
```
Comment (lumberj) [inline src/api/ledger/transactions.js]: What does this filter do?

Comment (clark800) [inline src/api/ledger/transactions.js]: Checks that the transaction is after/before the "start" transaction

File: src/api/ledger/transactions.js (hunk 2 above)
Comment (clark800) [inline src/api/ledger/transactions.js]: `getTransaction` uses `this.remote`

**Comment by lumberj** [comment]
LGTM

**Pull request merged**